hail/python/hail/table.py

```diff
@@ -3408,7 +3408,7 @@ def multi_way_zip_join(tables, data_field_name, global_field_name) -> 'Table':
         global_field_name : :obj:`str`
             The name of the resulting global field
 
-        .. include:: _templates/experimental.rst
```

Comment:
We don't just want to delete this line. We want to move it up. See https://github.com/hail-is/hail/blob/e0a365b65c71dfcbe74d770ae81ad80518866836/hail/python/hail/methods/impex.py#L2175

The danger box here: https://hail.is/docs/0.2/methods/impex.html?highlight=import_gvcfs#hail.methods.import_gvcfs

is the effect it creates.